### ISSUE TITLE
Optimise inference time

### ISSUE THREAD

**Issue by truongnhat-ryo** [vinthony/video-retalking#22]
Thanks for the great work. you are all doing a great contribution to community.
However, is there any idea to reduce inference time, thanks,

Regards

**Comment by zhangziliang04** [comment]
报一下你现在硬件配置、视频尺寸和推理时间. 也许我可以给你一些建议。

**Comment by truongnhat-ryo** [comment]
Hi @zhangziliang04 
I use 1 GPU Tesla T4 16GB, video Avatar size 832x960 (width, height).
To generate 8s video, it took 390 second in total.

Any idea to improve this, thanks

**Comment by zhangziliang04** [comment]
这个项目，默认有一个缓冲机制，第二次同一个驱动视频，会服用第一次训练的结果，具体对应1-4步骤，你可以看具体看一下。

**Comment by truongnhat-ryo** [comment]
Thanks understood, Step 6 always took most of inference time, any idea to speed it up, regards.

**Comment by KartavyaBagga** [comment]
Yes, the Lip Synthesis (step 6) is taking 2 min for a 30s audio for 360x640 resolution video on an RTX 4090 24GB

**Comment by KartavyaBagga** [comment]
> Thanks understood, Step 6 always took most of inference time, any idea to speed it up, regards.

Did you find any solution for Speed ?

**Comment by AIhasArrived** [comment]
@KartavyaBagga  did you find yet?  same question for @truongnhat-ryo  @zhangziliang04 , how to make it faster ??

**Comment by marsjin** [comment]
前面几步是有缓存，第二次用统一形象会有节约时间
但是，第六步占总时间的70～80%左右，是优化的关键
还需要多研究研究这步
